### ISSUE TITLE
Feature request: Reel Counter

### ISSUE THREAD

**Issue by sukarsa** [openpnp/openpnp#25]
It would be nice to have part counter on each feeder to know how many parts are left on the reel.
So for example, we can set a feeder to have 5000 items, and it should get decremented as the machine runs.
At least before starting a job it should check if the counter is sufficient, otherwise raise a warning.

Perhaps a view of feeders items quantity would be interesting so before shopping we can take a look which reels needs reordering.

**Comment by vonnieda** [comment]
I'm closing old issues that have seen little interest. PRs welcome.
